internal/server/routes.go

```diff
@@ -268,15 +265,14 @@ type WellKnownJWKResponse struct {
 }
 
 func (a *API) wellKnownJWKsHandler(c *gin.Context) {
-	keys, err := access.GetPublicJWK(c)
+	rCtx := getRequestContext(c)
```

Comment:
We could further integrate this `wellKnownJWKsHandler` and make it one of our `HandlerFunc`. That's going to be a prerequisite for other changes, so I might try and do that here.

I also need to check if we have test coverage for this endpoint.

Reply:
I did this in the latest commit.